client/src/entrypoints/admin/telepath/widgets.js

```diff
@@ -201,17 +201,25 @@ class DraftailInsertBlockCommand {
     const result = window.draftail.splitState(
       window.draftail.DraftUtils.resetBlockWithType(editorState, 'unstyled'),
     );
-    // Run the split after a timeout to circumvent potential race condition.
-    setTimeout(() => {
-      if (result) {
-        this.split.fn(
-          result.stateBefore,
-          result.stateAfter,
-          result.shouldMoveCommentFn,
-        );
-      }
+    if (result.stateAfter.getCurrentContent().hasText()) {
```

Comment:
Just to check, does this pick up non-text additions - eg if I split directly before an atomic block like an image?

Reply:
Yep - I had the same doubt when I wrote it :-) But in my testing, it did indeed count images as text content - presumably because our images are actually represented in the Draftail state as a single space character with an entity applied to it (and the space does count as text).

Reply:
Awesome, happy to approve in that case - the rest looks good.